azure-pipelines.yml

```diff
@@ -0,0 +1,235 @@
+  - name: FileVersionPrefix
+    value: 16.9
+  - name: IncludeDjangoHtmlExtensions
+    value: true
+  - name: IncludeLiveShare
+    value: true
+  - name: IncludeMiniconda
+    value: true
+  - name: IncludeMSI
+    value: false
+  - name: IncludeUWP
+    value: false
+  - name: LocType
+    value: Full
+  - name: SignType
+    value: Real
+  - name: TeamName
+    value: DSTools
+  - name: TrackFileAccess
+    value: false
+  - name: VisualStudioVersion
+    value: 16.0
+  - name: VSTarget
+    value: 16.0
+
+  # This is used to suppress warnings from the auto-injected Nuget Security Analysis step.
+  # The internal packages we use that are not available externally all start with "Microsoft",
+  # which mitigates the package injection vulnerability, since nobody outside the company can publish them to nuget.org.
+  # See https://docs.opensource.microsoft.com/tools/nuget_security_analysis.html for more info.
+  - name: NugetSecurityAnalysisWarningLevel
+    value: none
+  
+  # PTVS variable group, for secrets
+  # This group contains an AzDO Personal Access Token (ArtifactServices.Symbol.PAT) that expires on 4/15/2021 which is used to publish symbols to symweb.
+  # If Visual Studio insertion PRs start failing the Symbol Check step, this token is most likely expired.
+  # See https://www.1eswiki.com/wiki/Using_Azure_DevOps_Symbols for more information about how to create a new PAT and how to update the variable group.
+  - group: PTVS
+
+# TODO:
+# For PR builds, don't real sign. Test sign instead.
+
+steps:
+
+# check out code clean from source control
+- checkout: self
+  clean: true
+
+# skip some steps if this is a PR build
+- ${{ if notin(variables['Build.Reason'], 'PullRequest') }}:
+
+  # add build tag
+  - powershell: 'Write-Host "##vso[build.addbuildtag]$env:VSTarget"'
+    displayName: 'Add vstarget build tag'
+
+  # install plugins needed for swixproj/vsmanproj, signing, and localization
+  - task: MicroBuildSwixPlugin@3
+    displayName: 'Install swix plugin'
+    inputs:
+      dropName: 'Products/$(System.TeamProject)/$(Build.Repository.Name)/$(Build.SourceBranchName)/$(Build.BuildNumber)'
+      dropServiceUri: 'https://vsdrop.corp.microsoft.com/file/v1'
+      feedSource: 'https://mseng.pkgs.visualstudio.com/DefaultCollection/_packaging/MicroBuildToolset/nuget/v3/index.json'
+
+  - task: MicroBuildSigningPlugin@3
+    displayName: 'Install signing plugin'
+    inputs:
+      signType: 'Test' # TODO: Change this to $(SignType) once Real signing is approved for this AzDO project ID
+      feedSource: 'https://devdiv.pkgs.visualstudio.com/DefaultCollection/_packaging/MicroBuildToolset/nuget/v3/index.json'
+
+  - task: MicroBuildLocalizationPlugin@3
+    displayName: 'Install localization plugin'
+    inputs:
+      type: $(LocType)
+      languages: 'VS'
+      feedSource: 'https://mseng.pkgs.visualstudio.com/DefaultCollection/_packaging/MicroBuildToolset/nuget/v3/index.json'
```

Comment:
this loc task says its referenced by a mircobuild task further down. did you find a doc on microbuild yaml?

Reply:
I don't understand the question. I just created yaml from the existing steps, I didn't change anything here.